README.md

```diff
@@ -0,0 +1,30 @@
+asb-installer
+======================
+
+An Ansible Playbook Bundle (APB) for deploying a the [Ansible Service Broker (ASB)](https://github.com/openshift/ansible-service-broker).
+
+## Requirements
+An OpenShift cluster. (e.g. `oc cluster up`)
+
+User must be logged in with cluster-admin privileges. (e.g. `oc login -u system:admin`)
+
+## Building the installer
+```bash
+apb build
+```
+
+## Installing Ansible Service Broker
```

Comment:
Are we supposed to be calling it the Ansible Broker now? It would be nice if you included an example on how to override the default values for something like the broker image.

Reply:
I went with Ansible Service Broker since https://github.com/openshift/ansible-service-broker still calls it that.  We really need to think about a downstream version of this tool.

Will add instructions for changing the installer image.